contracts/CharityChallenge.sol

```diff
@@ -61,6 +66,7 @@ contract CharityChallenge {
         address payable[] memory _npoAddresses,
         uint8[] memory _ratios,
         address _marketAddress,
```

Comment:
should we remove `_marketAddress`?

Reply:
We need to limit ABI changes. Also, this _marketAddress will be now pointing to the Realit.io contract.